src/platform/site-wide/sass/shame.scss

```diff
@@ -579,3 +579,16 @@ img[data-srcset] {
 .operating-status-title {
   margin: auto;
 }
+
+.operating-status-flag {
+  border-radius: 5px;
+}
+
+a.operating-status-link {
+  color: initial;
+  text-decoration: none;
+}
```

Comment:
Why shouldn't this look like a link if it is one?

Reply:
Just following the pattern indicated in the comp, @cvalarida: 
![image](https://user-images.githubusercontent.com/2404547/69345179-b573eb00-0c3e-11ea-9f8e-f4534c337c10.png)

Reply:
Updated pr with method #2, @cvalarida